core-bundle/composer.json

```diff
@@ -81,7 +81,7 @@
         "symfony/dom-crawler": "4.4.*",
         "symfony/event-dispatcher": "4.4.*",
         "symfony/expression-language": "4.4.*",
-        "symfony/filesystem": "4.4.*",
+        "symfony/filesystem": "^5.4",
```

Comment:
```suggestion
        "symfony/filesystem": "4.4.* || 5.4.*",
```
I would have just widened this requirement and kept using the webmozart lib, to maximize compatibility with extensions. But I’m fine either way, @contao/developers WDYT?

Reply:
IMHO we must get rid of the abandoned library anyways in the remaining lifetime of 4.9 (and so do all extension developers), so I'd vote for removing it.

Reply:
we never got rid of abandoned libraries in any LTS. We still use lexik and other stuff. There's no reason to change this 🤷

Reply:
Optionally allowing version 5.4 in this special case would be OK for me. Then our users could require `symfony/filesystem` in version 5 if no other extension explicitly requires version 4.